preview/infrastructure/harvester/provider.tf

```diff
@@ -0,0 +1,37 @@
+terraform {
+
+  backend "gcs" {
+    bucket = "3f4745df-preview-tf-state"
+    prefix = "preview"
+  }
+
+  required_version = ">= 1.2"
+  required_providers {
+    harvester = {
+      source  = "harvester/harvester"
+      version = ">=0.5.1"
+    }
+    k8s = {
+      source  = "hashicorp/kubernetes"
+      version = ">= 2.0"
+    }
+  }
+}
+
+# https://registry.terraform.io/providers/harvester/harvester/latest/docs
```

Comment:
I know we link to the docs in many places, but I find it bit redundant so would prefer to not have them unless there's a specific reason for it.

Reply:
👍